spec/readme.md

```diff
@@ -10,7 +10,7 @@ This tar file can be created from most common container tools (e.g., Docker).
 
 ## The base folder
 
-To be a Flywheel gear, the container in the tar file must include a folder named: `/flywheel/v0`.
+To be a Flywheel gear, the container in the tar file must include a folder named: `/flywheel/v0`.<br>
```

Comment:
And just in case you thought markdown was consistent, even within a single vendor... a carriage return *in a comment* turns into a line break, but *in a repo file* will not! :roll_eyes: